articles/libraries/auth0-swift/user-management.md

```diff
@@ -21,7 +21,7 @@ The `link` method accepts two parameters, the primary user id and the secondary
 
 ```swift
 Auth0
-   .users(token: "user token")
+   .users(token: accessToken)
```

Comment:
`accessToken` variable is not declared. Either declare it or use quotes `"access token"`. Applies to all changes on this file